CONTRIBUTING.md

```diff
@@ -1,28 +1,106 @@
+* [Testing Changes Locally](#testing-changes-locally)
+* [Previewing Changes to the Website](#previewing-changes-to-the-website)
+
+
+## Types of Tutorials
+
+Currently, we have notebook-based tutorials and script-based tutorials.
+Both types are referenced in this guide.
+Notebook-based tutorials act as walkthroughs of concepts
+(the [`spam` tutorial](https://github.com/snorkel-team/snorkel-tutorials/tree/master/spam) is a good example).
+Script-based tutorials act more as examples of how to deploy certain Snorkel functionality
+(the [`drybell` tutorial](https://github.com/snorkel-team/snorkel-tutorials/tree/master/drybell) is a good example).
+We default to using notebook-based tutorials.
+
+
+## Making Changes to an Existing Tutorial
+
+First, we recommend [posting an issue](https://github.com/snorkel-team/snorkel-tutorials/issues/new)
+describing the improvement or fix you want to make.
+Once you've worked out details with the maintainers, follow these general steps:
+
+1. Make your changes to the source files
+    * For notebook-based tutorials, we recommend making changes to the `.py` version
+    then syncing changes with `tox -e my_tutorial_dir -- sync`.
+    * For script-based tutorials, just make the changes as you normally would.
+1. [Test your changes locally](#testing-changes-locally)
+1. Submit a PR!
+
+## Adding a New Tutorial
+
+Before adding a new tutorial, we recommend posting a proposal to the
+[Snorkel community forum on Spectrum](https://spectrum.chat/snorkel/tutorials?tab=posts).
+Once you've worked out details with the maintainers, follow these general steps:
 
 1. Create a new top-level directory (e.g. `my_tutorial_dir`)
-2. Add a file called `.notebooks` to your tutorial directory and add the base name of each tutorial script/notebook pair (e.g. `my_tutorial`) as a separate line in `.notebooks`
-3. Add a `requirements.txt` to your directory if additional ones are needed
-4. Add a command to `[testenv]` in `tox.ini` by copying `spam` and add the `requirements.txt` file if necessary. Also add the command name to `envlist`.
-5. Write your tutorial either as a Python script (e.g. `my_tutorial_dir/my_tutorial.py`) in [Jupytext percent format](https://gist.github.com/mwouts/91f3e1262871cdaa6d35394cd14f9bdc) or a Jupyter notebook
-    * Run `tox -e my_tutorial_dir -- sync` to generate a notebook version from the Python script version. Run this command to update when changes are made to the tutorial script.
-    * Run `tox -e my_tutorial_dir -- sync --py` to generate a Python script version from the notebook version. Run this command to update when changes are made to the tutorial notebook.
-6. Run `tox -e my_tutorial_dir` to test out your tutorial. This is also the test that Travis CI will use to check compliance before merges.
+1. Add a tutorial configuration file to your tutorial directory
+    * For notebook-based tutorials, add a file called `.notebooks` to your tutorial directory
```

Comment:
Is the numbering off in the real thing too?

Reply:
nope! https://github.com/snorkel-team/snorkel-tutorials/blob/684738542a0a420021509812847280ea30a6a351/CONTRIBUTING.md#making-changes-to-an-existing-tutorial